--- a/share/server/core/defines/global.php
+++ b/share/server/core/defines/global.php
@@ -78,11 +78,11 @@ define('DEBUGFILE', '../../../var/nagvis
 // The last value wins.
 //
 // Path to the main configuration file
-define('CONST_MAINCFG', '../../../etc/nagvis.ini.php');
-define('CONST_MAINCFG_CACHE', '../../../var/nagvis-conf');
+define('CONST_MAINCFG', '/etc/nagvis/nagvis.ini.php');
+define('CONST_MAINCFG_CACHE', '/var/cache/nagvis/nagvis-conf');
 
 // Path to the main configuration conf.d directory
-define('CONST_MAINCFG_DIR', '../../../etc/conf.d');
+define('CONST_MAINCFG_DIR', '/etc/nagvis/conf.d');
 
 // The directory below the NagVis root which is shared by the webserver
 define('HTDOCS_DIR', 'share');
